File: Golang/1-Two Sum.go
func twoSum(nums []int, target int) []int {
    var results = []int{0, 0}
    for i, num_i := range nums {
        for j, num_j := range nums[i+1:] {
            if num_i == target-num_j {
                results[0] = i
                results[1] = j+i+1
                break
            }
        }
    }
    return results
}

File: Python3/1-Two Sum.py
class Solution:
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        for i, num_i in enumerate(nums):
            for j, num_j in enumerate(nums[i+1:]):
                if num_i == target-num_j:
                    return [i, j+i+1]
